packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx

```diff
@@ -118,6 +120,17 @@ const getDocs: () => JSX.Element = () => {
         <CustomStyleVideoGalleryExample />
       </Canvas>
 
+      <Heading>Local Video Camera Button</Heading>
+      <DetailedBetaBanner />
+      <Description>
```

Comment:
mention the prop that can be used to enable it in the description

Reply:
okay will add that!